src/gui/screen_print_preview.cpp

```diff
@@ -109,6 +111,12 @@ GCodeInfo::GCodeInfo() {
             sscanf(value_buffer, "%u", &filament_used_mm);
         } else if (name_equals("filament used [g]")) {
             sscanf(value_buffer, "%u", &filament_used_g);
+        } else if (name_equals("printer_model")) {
+            if (strncmp(value_buffer, "MINI", sizeof(value_buffer)) == 0) {
```

Comment:
/suggestion/ Instead of hardcoding the `"MINI"` value, add it to the printer's configuration (if it's not there already).

Reply:
commit `e5f0c48`